src/components/Header/components/HeaderInfo/HeaderInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from "react";
+import { NavLink } from "react-router-dom";
+import cn from "classnames";
+import { ABOUT_INFO_LINKS, CONTACTS_INFO_LINKS } from "@/common/constants";
+import { NavigationLink } from "@/common/types";
+import styles from "./HeaderInfo.module.scss";
+
+export const HeaderInfo: FC = () => {
+  return (
+    <nav className={styles.headerInfoContainer}>
+      <ul className={styles.headerInfoList}>
+        {CONTACTS_INFO_LINKS.map((contact: NavigationLink) => (
+          <li className={styles.headerInfoListItem} key={contact.id}>
+            <NavLink to={contact.link}>{contact.title}</NavLink>
+          </li>
+        ))}
+      </ul>
```

Comment:
Can we omit this info on mobile ? Isn't it important?

Reply:
I think yes because all of this info is in the footer

![01](https://github.com/vynnykpff/reenbit-freshnesecom-app/assets/100035730/ca0b497f-5228-4f16-9062-3887cfa83a39)